config/jobs/cert-manager/go-pkcs12/go-pkcs12-presubmits.yaml

```diff
@@ -0,0 +1,25 @@
+presubmits:
+  cert-manager/go-pkcs12:
+
+    - name: pull-cert-manager-go-pkcs12-verify
+      always_run: true
+      max_concurrency: 8
+      decorate: true
+      branches:
+        - master
+      spec:
+        containers:
+          - image: europe-west1-docker.pkg.dev/cert-manager-tests-trusted/cert-manager-infra-images/golang-dind:20250327-a3af8ba-1.24
+            args:
+              - runner
+              - make
```

Comment:
@ali-hamza-noor, I don't think there is a Makefile in that repo.

Reply:
Merged now, so we have a Makefile.